render/internal/context/imagerender/context.go

```diff
@@ -212,7 +212,7 @@ func (dc *Context) SetRGB255(r, g, b int) {
 	dc.SetRGBA255(r, g, b, 255)
 }
 
-// SetRGBA sets the current color. r, g, b, a values should be between 0 and 1,
+// SetNRGBA sets the current color. r, g, b, a values should be between 0 and 1,
```

Comment:
Seems incorrect.  Check golint.
Seems like should be SetRGBA..

Reply:
Right solved.